### PR TITLE
02 sample app/12/05 edit training implementation

### LINKED ISSUE #12: Create Sample 02 Sample App\05 EditTraining
Step by step guided sample.

Edit training, include validation.

Simple admin edit.

### PR DESCRIPTION
**Pull request by nasdan**
This PR close #12 

- I create sample 06 for validations

### CODE REVIEW

File: 02 Sample App/00 Boilerplate/package.json [hunk 1]
```diff
@@ -43,10 +43,10 @@
     "bootstrap": "^3.3.7",
     "core-js": "^2.4.1",
     "jquery": "^3.1.1",
-    "lc-form-validation": "^0.1.7",
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
     "react-router": "^3.0.2",
-    "toastr": "^2.1.2"
+    "toastr": "^2.1.2",
+    "validate.js": "^0.11.1"
```
Comment (brauliodiez) [inline 02 Sample App/00 Boilerplate/package.json]: If it's just edit training and no validation should we remove validate.js? and add the step later on?

File: 02 Sample App/00 Boilerplate/readme.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Summary steps:
     - [Bootstrap](https://github.com/twbs/bootstrap).
     - [React and ReactDOM](https://github.com/facebook/react).
     - [React router](https://github.com/reacttraining/react-router) for navigation.
-    - [lc-form-validation](https://github.com/Lemoncode/lcFormValidation) for validations.
+    - [validate.js](https://github.com/ansman/validate.js) for validations.
```
Comment (brauliodiez) [inline 02 Sample App/00 Boilerplate/readme.md]: Same with the readme?

File: 02 Sample App/03 TrainingListA/webpack.config.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ module.exports = {
       'react',
       'react-dom',
       'react-router',
-      'lc-form-validation',
+      'validate.js',
```
Comment (brauliodiez) [inline 02 Sample App/03 TrainingListA/webpack.config.js]: Same here

**Comment by brauliodiez** [comment]
Good stuff, just found the validate.js thingy that I think could be move the plumbing to next sample?

File: 02 Sample App/00 Boilerplate/package.json (hunk 1 above)
Comment (nasdan) [reply 02 Sample App/00 Boilerplate/package.json]: Ok

**Pull request merged**